### PR TITLE
Extend LiveDashboard

### PR DESCRIPTION
**Pull request by rorymckinley**
* Enable Db Information
* Enable Os Information

## Notes for the reviewer

This PR makes it possible for the LiveDashboard to display OS- and DB-related data.

I am not sure how useful the OS-level data would be in a containerised environment but I think it would be useful
in situations where the infrastructure is either a metal box or a VM.

image

The DB information will, I think, be more generally useful when it becomes time to answer the question: 'Why does the database hate me?'.

image

## Related issue

Fixes #

## Review checklist

- [x] I have performed a **self-review** of my code
- [ ] I have verified that all appropriate **authorization policies** have been implemented and tested
- [ ] If needed, I have updated the **changelog**
- [ ] Product has **QA'd** this feature

### CODE REVIEW

**Comment by rorymckinley** [comment]
It appears that the code coverage failure is due to enabling `os_mon`. Will dig a bit and see if it can be resolved.

image

**Comment by rorymckinley** [comment]
Turns out my previous diagnosis was incorrect - original failure was caused by a flickering test. When I fixed that, I exposed another flickering test and, in fixing that, another flickering test surfaced. So, I am making this PR draft until I can fix the last test.

File: test/lightning_web/live/project_live_test.exs [hunk 1]
```diff
@@ -508,7 +508,14 @@ defmodule LightningWeb.ProjectLiveTest do
           ~p"/projects/#{project.id}/settings#vcs"
         )
 
-      Process.sleep(15)
+      Enum.take_while(0..30, fn _index ->
```
Comment (rorymckinley) [inline test/lightning_web/live/project_live_test.exs:511]: I would have preferred something that could have looked at some underlying state to determine when to break out of the loop but (within the constraints of my limited Elixir) - it appears that an unsupervised Task runs and then passes the state to the Live view - so it does not look like there is something other than the rendered view that the test can reach into.

**Comment by rorymckinley** [comment]
The case for the last flickering test () is less [clear cut](https://github.com/rorymckinley/Lightning/blob/main/test/lightning_web/live/work_order_live_test.exs#L422). I can't find an obvious element that could result in the test flickering - there does not appear to be an obvious asynchronous component.

So, going to leave this for now.

**Review by taylordowns2000 — APPROVED** [review]

**Pull request merged**